source/collaborate/access-your-workspace.rst

```diff
@@ -43,6 +43,9 @@ Mattermost is accessible through a URL link. You'll receive a link from your Mat
     3. Enter your user credentials to log into Mattermost. 
     4. The team that displays first in the team sidebar opens. If you're not a member of a team yet, you're prompted to select a team to join.
 
+    .. note::
+
+      Depending on how your user credentials are set up in Mattermost, you may temporarily leave the desktop app during login while authenticating your credentials. Once you're successfully logged in to Mattermost, you'll be returned to the desktop app.
```

Comment:
I'd specifically mention external providers here and maybe cite a couple examples (like Google, Office 365). We can be more specific and say OAuth and SAML providers, but that might not be helpful for people.

Reply:
Thanks, @devinbinnie! Focusing the examples on well-known providers is a good customer-centric call. My latest commit includes your feedback. This PR is ready for your final review.